doc/cxl_ide_key_refresh.md

```diff
@@ -0,0 +1,13 @@
+# CXL.cachemem IDE Key Refresh
+This is a sample host software flow when it is refreshing the Keys/IVs associated with an active CXL.memcache IDE Stream.
+1. Host software sends ```CXL.Get_Key``` to EP if ```CXL.cachemem IV Generation Capable``` and ```CXL.cachemem IDE Key Generation Capable``` are set in the response of ```CXL.Query```. This is to generate the Keys/IVs of TX direction. Otherwise software shall generate the Keys/IVs of TX direction by itself.
+2. Host software generates the Keys/IVs of RX direction.
+3. Host software construts ```CXL.Key_Prog``` of TX direction with the Keys in step1. If IVs are generated by ```CXL.Get_Key```, software shall copies the IVs into ```CXL.Key_Prog``` and set ```Bit[3]: Use Default IV``` as ```0``` to indicate that Port shall use the Initial IV specified at ```Offset 13h+KSIZE```. Otherwise software shall set  ```Bit[3]: Use Default IV``` as ```1``` to indicate that Port shall use the ```Default IV construction```. After that software sends ```CXL.Key_Prog``` to EP.
```

Comment:
1. "CXL.xxx" => "CXL_IDE_KM.xxx".
2. "Bit[3]: Use Default IV" => "Key_Prog.Use_Default_IV"
3. "Default IV construction" => move quote. Not sure why.

Reply:
updated.